docs/Temperature sensors.md

```diff
@@ -11,6 +11,24 @@ Up to 8 can be connected to the flight controller.
 * Supply: 2.7 to 5.5V
 * Temperature range: -55 to +125°C
 
+On the purple LM75 (CJMCU-75), adress pins on the bottom of the PCB needs to be sorted to get it works (define i2c adress)
```

Comment:
I think sounds better:

"On the purple LM75 (CJMCU-75), address line pins on the bottom of the PCB need to be bridged either to ground or VCC."

Reply:
yep, that sound better and more informative 👍